prody/proteins/interactions.py

```diff
@@ -1032,49 +1091,8 @@ def calcHydrogenBondsTrajectory(atoms, trajectory=None, **kwargs):
         selection='chain A and resid 1 to 50'
     If we want to study chain-chain interactions:
         selection='chain A', selection2='chain B' """
-
-    try:
-        coords = (atoms._getCoords() if hasattr(atoms, '_getCoords') else
-                    atoms.getCoords())
-    except AttributeError:
-        try:
-            checkCoords(coords)
-        except TypeError:
-            raise TypeError('coords must be an object '
-                            'with `getCoords` method')
-
-    HBs_all = []
-    start_frame = kwargs.pop('start_frame', 0)
-    stop_frame = kwargs.pop('stop_frame', -1)
-
-    if trajectory is not None: 
-        if isinstance(trajectory, Atomic):
-            trajectory = Ensemble(trajectory)
-        
-        nfi = trajectory._nfi    
-        trajectory.reset()
-        numFrames = trajectory._n_csets
-        
-        for j0, frame0 in enumerate(trajectory[start_frame:], start=start_frame):
-            if (j0 > 0 and j0 > stop_frame) or (j0 < 0 and j0+numFrames > stop_frame): 
-                break
-  
-            LOGGER.info('Frame: {0}'.format(j0))
-            protein = atoms.select('protein')
-            hydrogen_bonds = calcHydrogenBonds(protein, **kwargs)
-            HBs_all.append(hydrogen_bonds)
-        trajectory._nfi = nfi
     
-    else:
-        if atoms.numCoordsets() > 1:
-            for i in range(len(atoms.getCoordsets()[start_frame:stop_frame])):
-                LOGGER.info('Model: {0}'.format(i+start_frame))
-                atoms.setACSIndex(i+start_frame)
-                protein = atoms.select('protein')
-                hydrogen_bonds = calcHydrogenBonds(protein, **kwargs)
-                HBs_all.append(hydrogen_bonds)
-        else:
-            LOGGER.info('Include trajectory or use multi-model PDB file.')
+    HBs_all = calcInteractionsMultipleFrames(atoms, 'HBs', trajectory, **kwargs)
```

Comment:
you could save on a variable and just say 
`return calcInteractionsMultipleFrames(atoms, 'HBs', trajectory, **kwargs)`